README.rst

```diff
@@ -12,112 +12,66 @@ The list of checkers is documented on the Mozilla wiki:
 https://wiki.mozilla.org/Release_Management/autonag
```

Comment:
@calixteman - as long as you're updating the docs, want to add info about the required format  for names in the round robin calendars [here](https://github.com/mozilla/relman-auto-nag#setting-up-round-robin-triage-rotations)? I came in here planning to do this, but since you've already got this PR open I figured maybe just roll it in? Otherwise I'm happy to open a separate PR if you'd prefer.